internal/docstore/dynamodocstore/query.go

```diff
@@ -65,6 +66,9 @@ func (c *collection) checkPlan(qr *queryRunner) error {
 	return nil
 }
 
+// From https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.html:
```

Comment:
Is this comment intended to be not attached with any code block?

Reply:
Removed in the next PR.